elytron/src/main/java/org/wildfly/extension/elytron/AuditResourceDefinitions.java

```diff
@@ -240,14 +240,16 @@ static ResourceDefinition getRotatingFileAuditLogResourceDefinition() {
                     final SecurityEventVisitor<?, String> formatter = Format.JSON == format ? JsonSecurityEventFormatter.builder().setDateFormatSupplier(bind(SimpleDateFormat::new, DATE_FORMAT)).build() : SimpleSecurityEventFormatter.builder().setDateFormatSupplier(bind(SimpleDateFormat::new, DATE_FORMAT)).build();
                     AuditEndpoint endpoint;
                     try {
-                        endpoint = RotatingFileAuditEndpoint.builder()
+                        FileAuditEndpoint.Builder builder = RotatingFileAuditEndpoint.builder()
```

Comment:
Not really, the cast was necesary because the builder chain returns a FileAuditEndpoint, but since the order in the builder chain order doesn't matter, we can change it to avoid the cast.
I'm going to do it